### ISSUE TITLE
Consider adding information to code snippets to say what the code does

### ISSUE THREAD

**Issue by kimdurante** [sul-dlss/earthworks#1373]
It might be useful to add information about what the Python, R, and Leaflet code snippets do, i.e Import Data, Download Data, etc...

**Comment by marlo-longley** [comment]
Inspiration (the labels)

<img width="665" alt="Screenshot 2024-09-06 at 12 24 32" src="https://github.com/user-attachments/assets/3f964a5b-d70b-4227-9be5-0ef762fb7134">

**Comment by mapninja** [comment]
I agree, but I'm not sure it needs to be a UI element, rather than simply using inline code comments? That way updates to the code don't require UI updates, as well?
